distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -127,16 +122,17 @@ from project.Data.Boolean import all
 
         The equality check of two objects:
         - verifies both objects share the same type of `comparator`
-        - consults their `compare`, or `equals` method, based on whether the comparator is
-          ordered or unordered.
-        - if the result is `Ordering.Equal` the system also checks that both instances have the same `hash`
-        - the `hash` code check may be done only from time to time to speed things up
+        - consults their `compare` method.
+        - if the result is `Ordering.Equal` the system may also checks that both
```

Comment:
```suggestion
        - if the result is `Ordering.Equal` the system may also check that both
```
don't really know what "system" means in this context. I would avoid adding new terminology

Reply:
I got rid of the whole paragraph "The equality check of two objects", since it is quite outdated anyway.